tests/calculate.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cd $1
+
+find . -name '*.vcf'  | xargs md5sum 
```

Comment:
I would recommend omitting header lines: ` for f in $(find . -xtype f -name "*.vcf.gz" | sort -V);do zcat $f | grep -v ^# | md5sum`